src/commands/commands.js

```diff
@@ -42,8 +42,11 @@ Office.onReady(() => {
 async function merge(event) {
   openMergeTaskpane()
   window.app.setTaskpaneStatus(taskpaneStatus.MERGE);
-  window.app.setMergeState(mergeState.MERGE_IN_PROGRESS);
+
+  // update UI and execute merge
+  window.app.setMergeState({status: mergeState.MERGE_IN_PROGRESS, conflicts: null});
```

Comment:
Gonna open an issue to unify the language around "state" and "status." They were original the same type of state but now no longer.